gammapy/spectrum/simulation.py

```diff
@@ -70,12 +70,19 @@ def npred_background(self):
 
         Calls :func:`gammapy.spectrum.utils.CountsPredictor`.
         """
-        predictor = CountsPredictor(livetime=self.livetime,
+        if self.background_model is None:
```

Comment:
Can you quickly explain, what this does? How can you estimate `npred_background` if the background model is `None`?